src/datatrove/pipeline/filters/fasttext_filter.py

```diff
@@ -77,7 +77,7 @@ def model(self):
             model_file = os.path.join(download_dir, strip_protocol(self.model_url).replace("/", "_"))
             if not os.path.isfile(model_file):
```

Comment:
this is not consistent with the comment within the lock, i.e. running the check after acquiring the lock. I suppose ideally there should be a single line here which would be a call to a function that would do everything from checking if the file exists, to downloading it with the lock and printing the messages

Reply:
mb